InvenTree/plugin/base/label/test_label_mixin.py

```diff
@@ -75,13 +74,10 @@ class WrongPlugin(LabelPrintingMixin, InvenTreePlugin):
 
     def test_installed(self):
         """Test that the sample printing plugin is installed."""
-        # Get all label plugins
-        plugins = registry.with_mixin('labels')
-        self.assertEqual(len(plugins), 2)
 
-        # But, it is not 'active'
+        self.do_activate_plugin()
         plugins = registry.with_mixin('labels')
-        self.assertEqual(len(plugins), 1)
+        self.assertEqual(len(plugins), 2)
```

Comment:
So activation is not tested anymore?

Reply:
There are plenty of other tests for plugin activation, either via the API or directly called via the registry.

Reply:
we had problems with this not working for label plugins in the past but fine for me if you are confident the other test catch that

Reply:
This refactor has uncovered a few areas where the plugins were not behaving as "expected" - I will definitely ensure that the unit tests are up to scratch :)

Reply:
At this point this is a full refactor, I will have to retest and rewrite everything in my repos anyway.